docs/content/users/install/ddev-installation.md

```diff
@@ -97,57 +97,67 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
+    You can install on Windows three different ways: Using WSL2 with Docker inside, Using WSL2 with Docker Desktop, and installing directly on traditional Windows with an installer (not recommended). Each is described below.
 
-    **WSL2 is the recommended installation method for all Windows users**.
+    Windows WSL2 is the best way to run DDEV and your web components. It’s Linux, which means a different experience for some Windows users. It’s Ubuntu Linux by default as described here, so it’s worth taking a little time to explore how Ubuntu and Bash work, including standard system commands and installation and upgrade procedures.
 
-    **Using WSL2 to install and run DDEV is not the same as using Docker Desktop's WSL2 engine, which itself runs in WSL2, but can serve applications running in both traditional WIndows and inside WSL2.**
+    ### Things You Need to Know about WSL2 and DDEV
 
-    **All Windows 10/11 editions (including Windows 10 Home) support WSL2**. If you’re already familiar with DDEV on Windows, you might have been using NFS for better filesystem performance. **You won't need NFS anymore once you switch to WSL2**, since it provides awesome filesystem performance out of the box.
+    * **WSL2 is the recommended installation method for all Windows users.** All Windows 10/11 editions (including Windows 10 Home) support WSL2. The performance is way, way faster in every way, and you're less likely to have obscure Windows problems if you use WSL2.
+    * WSL2 Projects go in `/home`, not on the Windows filesystem. Make sure you put your projects in the Linux filesystem (e.g. `/home/<your_username>`), **not** in the Windows filesystem (`/mnt/c`), because you’ll get vastly superior performance on the Linux filesystem. If you put your project in `/mnt/c` you will not get the benefits of WSL2.
+    * WSL2 hostname management: DDEV attempts to manage custom hostnames via the Windows-side hosts file (usually `C:\Windows\system32\drivers\etc\hosts`) rather than the hosts file inside WSL2. It can only do this if DDEV (with sudo.exe) is installed and in the `$PATH` on the Windows side. (DDEV inside WSL2 uses `ddev.exe` on the Windows side as a proxy to update the Windows hosts file.) If `ddev.exe --version` shows the same version as `ddev --version` you're all set up. Otherwise, install DDEV on Windows using `choco upgrade -y ddev` or by downloading and running the Windows installer. (The scripts below install DDEV on the Windows side, so you won't need to do anything if you've used one of those to install.) (If the Windows escalation happens to often to you, you can calm it down with `gsudo.exe  cache on` and `gsudo.exe config CacheMode auto`, see [gsudo docs](https://github.com/gerardog/gsudo#credentials-cache).
```

Comment:
This one’s much harder to whittle down, and one of the reasons Windows should have its own page and/or a section in [How DDEV Works](https://ddev.readthedocs.io/en/stable/users/basics/architecture/). It really is a different beast, and it seems like it’d be intimidating for a newcomer—like the usual barrier of understanding containerized environments compounded by this miraculous Linux-in-Windows phenomenon.

I did what I could with this:

```suggestion
    * Custom hostnames are still managed on the Windows side, not within WSL2.  
      DDEV attempts to manage custom hostnames via the Windows-side hosts file, usually at `C:\Windows\system32\drivers\etc\hosts`. It can only do this if DDEV (with `sudo.exe`) is installed and in the Windows-side `$PATH`. (DDEV inside WSL2 uses `ddev.exe` on the Windows side as a proxy to update the Windows hosts file.) If `ddev.exe --version` shows the same version as `ddev --version` you’re all set up. Otherwise, install DDEV on Windows using `choco upgrade -y ddev` or by downloading and running the Windows installer. (The WSL2 scripts below install DDEV on the Windows side, taking care of that for you.) If the Windows escalation happens too often to you, you can calm it down by running `gsudo.exe cache on` and `gsudo.exe config CacheMode auto`, see [gsudo docs](https://github.com/gerardog/gsudo#credentials-cache).
```

But I still have questions:

1. “It can only do this if DDEV (with `sudo.exe`) is installed”—can this be more clear? Did we install DDEV’s binary using `sudo.exe`? Do I need to install DDEV and another app called `sudo.exe`? I have no idea what this actually means. (And I’m Windows-incompetent, so sorry if this should have been obvious!)
2. DDEV attempts to manage custom hostnames on the Windows side, but what happens if its attempt fails? What am I risking if I ignore this bullet point? (This may be a topic for another page where it’s not in the way of installation instructions.)
3. What is the “Windows escalation”? As a reader at this point I’m looking at installing DDEV, presumably for the first time, and that’s not a phrase that’s been used previously and certainly not something I would have run into yet.
4. This is a hefty bullet point, and it seems to me that I can mostly ignore it if I’m using one of the WSL2 installers and/or my `ddev.exe --version` and `ddev --version` versions match. Is this all relevant and essential to someone that’s about to install DDEV via WSL2?

Reply:
Thanks for the questions!
1. The reality is sudo is installed with DDEV, so this can be simpler. Just say "DDEV installed on Windows side"
2. If attempt fails you get a simple error message that explains the situation. Maybe not as simple as we'd hope.
3. "Windows escalation" should be "Windows UAC Escalation", which is where the big screen comes up in windows and asks if it's OK to do. 
4. I agree this is not particularly important if using the script to install. But this is a massive improvement to some people's WSL2 experience, a problem that has existed since we first started WSL2 support... and as we see here... it introduces its own complexity.